demo.sh

```diff
@@ -125,8 +128,8 @@ spec:
     repoURL: http://$GOGS_HOSTNAME/gogs/spring-petclinic-config
 EOF
   oc apply -k argo -n $cicd_prj
-  oc policy add-role-to-user admin system:serviceaccount:$cicd_prj:argocd-argocd-application-controller -n $dev_prj
-  oc policy add-role-to-user admin system:serviceaccount:$cicd_prj:argocd-argocd-application-controller -n $stage_prj
+  oc policy add-role-to-user admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller -n $dev_prj
```

Comment:
Is the Argo CD instance form the cicd namespace replaced with the one in the openshift-gitops space?